tests/utilities.py

```
@@ -18,47 +18,53 @@ def stub_async_write_ha_state(entity: Any) -> None:
 
     Args:
         monkeypatch: Pytest monkeypatch fixture.
-        module: Target module exposing `create_opnsense_client`.
+        module: Target module exposing client construction helpers.
         client_ctor: Callable/class used to construct fake clients for tests.
     """
 
-    async def _create_opnsense_client(
+    def _create_opnsense_client(
         *,
+        hass: Any | None = None,
```

Ruff (0.15.17)

[warning] Dynamically typed expressions (typing.Any) are disallowed in `hass`

(ANN401)